### PR TITLE
uiTextField will now accept text input for the Alarm Title

### PR DESCRIPTION
**Pull request by antarv**
Alarm title can be added!!

### CODE REVIEW

File: AppleAlarm/AddViewController.swift [hunk 1]
```diff
@@ -35,7 +35,8 @@ class AddViewController: UIViewController {
         if segue.identifier == "DoneItem" {
             if let name = alarmTitleTF.text {
                 if !name.isEmpty {
-                    newItem = Alarm()
+                    newItem = Alarm(name: name, time: NSDate())
```
Comment (adm9958) [inline AppleAlarm/AddViewController.swift]: Don't forget to call alarmsManager.save() after you add this new Alarm object to the list of Alarms.

File: AppleAlarm/ListViewController.swift [hunk 2]
```diff
@@ -18,7 +18,7 @@ class ListViewController : UITableViewController {
     
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCellWithIdentifier("AlarmItemTableViewCell", forIndexPath: indexPath) as! AlarmItemTableViewCell
-        let item = itemsList[indexPath.row]
+        let item = alarmsManager.alarms[indexPath.row]
```
Comment (adm9958) [inline AppleAlarm/ListViewController.swift:21]: I thought I already made this change in one of my earlier merges?

Comment (adm9958) [inline AppleAlarm/ListViewController.swift:21]: Never mind, you can keep this change in the file and deal with any merge conflicts if they occur.  LGTM.

**Comment by Billkaboo** [comment]
LGTM

**Pull request merged**